"""
Represents an exception when user authentication fails
"""
class CredentialsException(Exception):
    pass

"""
Represents an exception for when an entity is not found
"""
class NotFoundException(Exception):
    pass
